db/structure.sql

```diff
@@ -24,29 +24,6 @@ CREATE TYPE public.oban_job_state AS ENUM (
 );
 
 
---
--- Name: oban_jobs_notify(); Type: FUNCTION; Schema: public; Owner: -
---
-
-CREATE FUNCTION public.oban_jobs_notify() RETURNS trigger
```

Comment:
I'm curious why this changed on your PR since you didn't modify anything related to the schema?

Reply:
I did a `rake db:schema:dump` I think when I had the original structure.sql things were complaining about no users relation but I can check again